### PR TITLE
Simplify PR template checklist

### PR DESCRIPTION
**Pull request by fan-jin**
## Purpose
The existing checklist includes many redundant action items, leading to low utilization/value for the section. This PR looks to simplify it, and make explicit what needs to be done, in order to have the PR be eligible to be approved.

## Screenshots <sub><sup>(FE or BE changes that has corresponding FE)</sup></sub>

Screenshot 2023-12-06 at 7 07 22 AM

## Testing Setup Instructions <sub><sup>(what setup is required in order to test your changes, and how you tested them)</sup></sub>

Make a PR and observe the template is being utilized

## What Could Go Wrong <sub><sup>(and how you verified that it won't)</sup></sub>

The  PR template is not utilized due to overrides from repos, this has been validated.

## Documentation <sub><sup>(that requires updating in Notion / Github)</sup></sub>

This is itself a documentation

## Post-release Monitoring Plan <sub><sup>(what's your plan to monitor the changes to make sure they have the intended effect)</sup></sub>

Monitor for new PRs opened in organization repos to utilize this updated PR template

## Checklist
- [x] Add a brief description of the purpose of this PR.
- [x] Add screenshots for this PR.
- [x] Add testing instructions for this PR.
- [x] Introduce or modify relevant documentation
- [x] Ensure unhappy paths are considered and tested
- [x] Ensure there is a plan for monitoring post-deployment
- [ ] Test your changes in the deployed PR preview environment / [run migration locally](https://github.com/clearbanc/code/tree/staging/packages/migrations#run-migrations).
- [ ] Ensure test coverage value in config file match actual coverage (if applicable).

## Jenkins Build Triaging
To rebuild a this PR in Jenkins comment in this PR with: `@Jenkins rebuild`.
To unfreeze frozen builds try to close and reopen the PR.
For further help please reach out to `@infra-help` in `#devops` slack channel.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -11,20 +11,13 @@
 ## Post-release Monitoring Plan <sub><sup>(what's your plan to monitor the changes to make sure they have the intended effect)</sup></sub>
 
 ## Checklist
-- [ ] Update PR title to follow Conventional Commits/Semantic Versioning
-  - For more information read [What should my PR Title be](https://github.com/clearbanc/code/tree/staging/packages#what-should-my-pr-title-be)
 - [ ] Add a brief description of the purpose of this PR.
 - [ ] Add screenshots for this PR.
 - [ ] Add testing instructions for this PR.
```
Comment (gloriousducks) [inline .github/pull_request_template.md]:
```suggestion
- [ ] Add a brief description, screenshots (if applicable), and testing instructions for this PR.
```

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -11,20 +11,13 @@
...
 - [ ] Introduce or modify relevant documentation
-- [ ] Apply appropriate label (e.g. `needs review`).
-- [ ] Remove changes that aren't related to the purpose of this PR.
-- [ ] Remove debugging code that was accidentally committed.
 - [ ] Ensure unhappy paths are considered and tested
-- [ ] Ensure there is a plan for monitoring post deployment
-- [ ] Ensure the build is passing.
-- [ ] Test your changes in the deployed PR preview environment / [run migration locally](https://github.com/clearbanc/code/tree/staging/packages/migrations#run-migrations).
-- [ ] Leave pre-emptive comments on changes you'd like to discuss.
+- [ ] Ensure there is a plan for monitoring post-deployment
```
Comment (gloriousducks) [inline .github/pull_request_template.md]: ```- [ ] Ensure unhappy paths are considered/tested, and there is a plan for monitoring post-deployment```

Comment (gloriousducks) [reply .github/pull_request_template.md]: consolidate the two WCGW and monitoring

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -11,20 +11,11 @@
...
-  - For more information read [What should my PR Title be](https://github.com/clearbanc/code/tree/staging/packages#what-should-my-pr-title-be)
-- [ ] Add a brief description of the purpose of this PR.
-- [ ] Add screenshots for this PR.
-- [ ] Add testing instructions for this PR.
+- [ ] Add a brief description, screenshots (if applicable), and testing instructions for this PR.
 - [ ] Introduce or modify relevant documentation
-- [ ] Apply appropriate label (e.g. `needs review`).
-- [ ] Remove changes that aren't related to the purpose of this PR.
-- [ ] Remove debugging code that was accidentally committed.
 - [ ] Ensure unhappy paths are considered and tested
```
Comment (timbuckiii) [inline .github/pull_request_template.md]:
```suggestion
```

**Review by timbuckiii — APPROVED** [review]

**Review by gloriousducks — APPROVED** [review]

**Pull request merged**